lib/charms/ovn_charm.py

```diff
@@ -1328,13 +1335,38 @@ def configure_bridges(self):
                                            portdata={
                                                'external-ids': {
                                                    'charm-ovn-chassis': br}})
+            # We can use the MAC of the bridge as the unique chassis MAC. If
+            # a physical port has been added to the bridge, the MAC will most
+            # likely be inherited. If not, a unique mac will be generated
+            # anyway.
+            hw_addr = None
+            try:
+                hw_addr = ch_host.get_nic_hwaddr(br)
+            except Exception as err:
+                ch_core.hookenv.log('failed to get MAC for bridge "{}": {}'
+                                    .format(br, err),
+                                    level=ch_core.hookenv.ERROR)
+            if hw_addr:
+                networks = ovnbridges.get(br, None)
```

Comment:
nit: If you make this get(br, []) you don't need the 'if networks' check below, can just iterate

networks = ovnbridges.get(br, [])
for net in networks:

Reply:
that's a great point. Changing.